test/e2e/ClientMode.test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const testServer = require('../helpers/test-server');
+const config = require('../fixtures/client-config/webpack.config');
+const runBrowser = require('../helpers/run-browser');
+
+describe('clientMode', () => {
+  describe('sockjs', () => {
+    beforeAll((done) => {
+      const options = {
+        port: 9000,
```

Comment:
Sorry but I change tests system. See https://github.com/webpack/webpack-dev-server/pull/2005.
And I'll merge https://github.com/webpack/webpack-dev-server/pull/2007 asap. Please change 🙏

Reply:
@hiroppy done